### ISSUE TITLE
unnecessary public constant in TextResource

### ISSUE THREAD

**Issue by simonjenga** [jcabi/jcabi-xml#102]
The ```public constant``` in the class ```com.jcabi.xml.TextResource.ENCODING``` is unnecessary.

```
/**
 * Encoding.
 */
public static final String ENCODING = "UTF-8";
```

**Comment by dmarkov** [comment]
@yegor256 please do something about this issue

**Comment by yegor256** [comment]
@simonjenga fixed in 821fe23, thanks!

**Comment by simonjenga** [comment]
@yegor256 We should always try to minimize the scope of variables as much as possible. The constants now being global are available throughout the whole class BUT are only being used once.
Other methods shouldn't see them because it breaks encapsulation.

Another thing is memory. When you define it as a static it will be in program memory all the time.

Instead of a local constant field we should use the literal value ```"UTF-8"``` inside the methods.

After these methods are executed these literal values get destroyed.

http://stackoverflow.com/questions/24551989/string-constant-vs-variable-in-a-java-method
http://stackoverflow.com/questions/27031300/method-specific-constants-in-java

**Comment by yegor256** [comment]
@simonjenga very good point! fixed in 9c06484

**Comment by simonjenga** [comment]
@yegor256  :thumbsup: :ok:
